app/views/eventRegistrations.html

```diff
@@ -72,22 +72,22 @@
       <th class="tablecenter" width="160"><a href="" ng-click="setOrder('created')">Started</a></th>
       <th class="tablecenter" width="160"><a href="" ng-click="setOrder('completed')">Completed</a></th>
       <th ng-show="eventHasCost()" class="tablecenter" width="60">Paid</th>
-      <th class="tablecenter" width="100">Options</th>
+      <th class="tablecenter" width="160">Options</th>
       <th width="30"></th>
     </tr>
   </thead>
   <tbody ng-repeat="r in (regFilter = (registrants | filter:searchFilter | filter:paymentStatus | filter:{registrantTypeId: filterRegistrantType} | filter:completeStatus)) | orderBy:answerSort:reversesort">
     <tr ng-class="{'active': $even}" class="noselect">
-      <td ng-click="expandRegistration(r.id)">{{r.firstName}} {{r.lastName}}</td>
-      <td ng-click="expandRegistration(r.id)">{{r.email}}</td>
-      <td ng-repeat="block in blocks | filter:{ visible: true }" ng-click="expandRegistration(r.id)">
+      <td ng-click="expandRegistration(r.id)" ng-class="{'strike-through':r.withdrawn}">{{r.firstName}} {{r.lastName}}</td>
+      <td ng-click="expandRegistration(r.id)" ng-class="{'strike-through':r.withdrawn}">{{r.email}}</td>
+      <td ng-repeat="block in blocks | filter:{ visible: true }" ng-click="expandRegistration(r.id) "ng-class="{'strike-through':r.withdrawn}">
```

Comment:
The ng-click value has an extra space at the end.  The ng-class attribute needs a space before.